src/WorkingStrings.java

```diff
@@ -0,0 +1,18 @@
+public class WorkingStrings {
+
+    public static void main(String[] args) {
+
+        String s = "Hello World!";
+
+        System.out.println("Länge: " + s.length());
+        System.out.println("In Gross: " + s.toUpperCase());
+        System.out.println("In klein: " + s.toLowerCase());
+        String newWord = "Coders Bay";
```

Comment:
was ich hier gemeint hatte war eher in der Variable `s` das World mit Codersbay auszutauschen. hierfür sinnvoll wär die `replace` Methode